core/src/Cache/CacheFallbackOperations.cs

```diff
@@ -38,7 +38,7 @@ internal class CacheFallbackOperations
     {
         public static void WriteMsalRefreshToken(ITokenCacheAccessor tokenCacheAccessor,
             AdalResultWrapper resultWrapper, string authority, string clientId, string displayableId,
-            string identityProvider, string givenName, string familyName, string objectId)
+             string givenName, string familyName, string objectId)
```

Comment:
`string identityProvider` wasn't being used anywhere in this method, so I just removed it. Doesn't make sense to pass in a parameter that is not used and `WriteMsalRefreshToken()` is only called here. We also don't have unit tests for this, but I need to learn more about the new cache format to write them.